docs/observability/notification.md

```diff
@@ -0,0 +1,54 @@
+|Data Scientist|Azure ML pipeline<br>-Success/Failure|-email<br>-Microsoft Teams<br>-Slack|- StorageQueue + LogicApp<br>- Directly set trigger with AML events e.g. LogicApp (See [document](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-use-event-grid))|-|
+|QA (Quality Assurance)|Azure ML pipeline<br>- Start/End of experiments<br>- Where the result is (i.e. Azure Dashboard)|-email<br>-Microsoft Teams<br>-Slack |- StorageQueue + LogicApp<br>- Directly set trigger with AML events e.g. LogicApp (See [document](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-use-event-grid))<br>*QA team does not have access to Azure DevOps and Azure ML|- [Trigger applications, processes, or CI/CD workflows based on Azure Machine Learning events (preview)](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-use-event-grid)<br> - [Azure Machine Learning events (preview) section in Spike wiki - Notification](/Spikes/Notification)|
+|Business Analyst|Cost management|-|- Azure Cost Management|[Azure Cost Management](https://docs.microsoft.com/en-us/azure/cost-management-billing/cost-management-billing-overview)|
+
+## Azure DevOps monitoring and notification
+## Personal
+### Current Setting
+Open your profile menu, and choose Notifications,
+
+![Azure-Devops-monitoring-and-notification-01.png](../media/observability/Azure-Devops-monitoring-and-notification-01.png)
+
+Or Notification settings.
+
+![Azure-Devops-monitoring-and-notification-02.png](../media/observability/Azure-Devops-monitoring-and-notification-02.png)
+
+The profile menu appears as shown below based on whether the New Account Manager feature has been enabled or not.
+
+![Azure-Devops-monitoring-and-notification-03.png](../media/observability/Azure-Devops-monitoring-and-notification-03.png)
+## Team, Project
+### Current Setting
+You can see the current **team or project level** notifications setting from Project Settings in Azure DevOps.
+
+### Add new notifications
+1. Click "New subscription" and select a templete
+![Azure-Devops-monitoring-and-notification-04.png](../media/observability/Azure-Devops-monitoring-and-notification-04.png)
```

Comment:
This image doesn't render.

Reply:
@liupeirong Thank you for finding out the render issue. Just fixed it.